src/modules/Auth/index.tsx

```diff
@@ -32,8 +32,7 @@ const AuthContainer = () => {
     });
     let payload = {
       user: {
-        name,
-        email,
+        token: credentialResponse.credential,
```

Comment:
credential is just the token or do we have other user information in it?

Reply:
Yes it is just the token, no any other information of user is there in it.